.github/workflows/jest-tests.yml

```diff
@@ -1,17 +1,36 @@
 name: Run Jest Tests
 
-# Triggers the workflow on push, pull request, or manual trigger (workflow_dispatch)
 on:
   push:
     branches:
-      - main  # or any branch you want the action to run on
+      - main
+      - main
```

Comment:
Is this duplication deliberate?

Reply:
Oh it's a mistake , must have been during the rebase